map/bookmark_helpers.cpp

```diff
@@ -292,7 +294,7 @@ std::string GetKMLPath(std::string const & filePath)
 {
   std::string const fileExt = GetFileExt(filePath);
   std::string fileSavePath;
-  if (fileExt == kKmlExtension)
+  if (fileExt == kKmlExtension || fileExt == kGpxExtension)
```

Comment:
Imported GPX should be converted/stored as KML. Is this change really needed?

Reply:
Can you please clarify the current behavior? Are imported GPX always converted and stored internally as KML files? Then there's no need to patch kml load code.

Reply:
I checked in debugger and looks like extension is checked on early stage of file loader where we still have .gpx:

![image](https://github.com/organicmaps/organicmaps/assets/128428520/fb7a4e1b-f953-4f64-8334-a93caac685e1)

Reply:
Am I correctly understood, that this part of the code is used _during the import_ too?

Reply:
Yes, it is executed when I import a track. Here is a stacktrace for this call.

```
[liborganicmaps.so] GetKMLPath(const std::__ndk1::basic_string<…> &) bookmark_helpers.cpp:295
[liborganicmaps.so] $_15::operator()() const bookmark_manager.cpp:1871
[liborganicmaps.so] std::__ndk1::__invoke<…>($_15 &) type_traits:3874
[liborganicmaps.so] std::__ndk1::__invoke_void_return_wrapper::__call<…>($_15 &) __functional_base:348
[liborganicmaps.so] std::__ndk1::__function::__alloc_func::operator()() functional:1557
[liborganicmaps.so] std::__ndk1::__function::__func::operator()() functional:1731
[liborganicmaps.so] std::__ndk1::__function::__value_func::operator()() const functional:1884
[liborganicmaps.so] std::__ndk1::function::operator()() const functional:2556
[liborganicmaps.so] base::thread_pool::delayed::ThreadPool::ProcessTasks() thread_pool_delayed.cpp:157
[liborganicmaps.so] std::__ndk1::__invoke<…>(void (base::thread_pool::delayed::ThreadPool::*&)(), base::thread_pool::delayed::ThreadPool *&) type_traits:3815
[liborganicmaps.so] std::__ndk1::__apply_functor<…>(void (base::thread_pool::delayed::ThreadPool::*&)(), std::__ndk1::tuple<…> &, std::__ndk1::__tuple_indices<…>, std::__ndk1::tuple<…> &&) functional:2853
[liborganicmaps.so] std::__ndk1::__bind::operator()<…>() functional:2886
[liborganicmaps.so] std::__ndk1::__invoke<…>(std::__ndk1::__bind<…> &) type_traits:3874
[liborganicmaps.so] std::__ndk1::__invoke_void_return_wrapper::__call<…>(std::__ndk1::__bind<…> &) __functional_base:348
[liborganicmaps.so] std::__ndk1::__function::__alloc_func::operator()() functional:1557
[liborganicmaps.so] std::__ndk1::__function::__func::operator()() functional:1731
[liborganicmaps.so] std::__ndk1::__function::__value_func::operator()() const functional:1884
[liborganicmaps.so] std::__ndk1::function::operator()() const functional:2556
[liborganicmaps.so] threads::SimpleThread::ThreadFunc(std::__ndk1::function<…> &&) thread.cpp:101
[liborganicmaps.so] std::__ndk1::__invoke<…>(void (*&&)(std::__ndk1::function<…> &&), std::__ndk1::__bind<…> &&) type_traits:3874
[liborganicmaps.so] std::__ndk1::__thread_execute<…>(std::__ndk1::tuple<…> &, std::__ndk1::__tuple_indices<…>) thread:273
[liborganicmaps.so] std::__ndk1::__thread_proxy<…>(void *) thread:284
[libc.so] __pthread_start(void*) 0x000000715d7a6cb4
[libc.so] __start_thread 0x000000715d744eb0
```